react/Footer/ToggleContainer/ToggleContainer.less

```diff
@@ -21,11 +21,6 @@
   transform-origin: 50% 50%;
 }
 
-.chevronSvg {
-  width: 12px;
-  height: 12px;
-}
```

Comment:
Now that we are changing the size of the chevron it no longer feels necessary to [translate it down a pixel](https://github.com/seek-oss/seek-style-guide/pull/408/files#diff-b5d3636b3c1d3de341bb87967b7051feR20), thoughts?

Reply:
Agree, removed